rewrites.js

```diff
@@ -219,7 +219,7 @@ exports.REWRITES = [
   },
   // New Routes -> New flow
   {
-    source: `/:collectiveSlug/:verb(donate)/:paymentMethod(crypto)?/:step(${contributionFlowSteps})?`,
+    source: `/:collectiveSlug/:verb(donate)/:donationMethod(crypto)?/:step(${contributionFlowSteps})?`,
```

Comment:
Not a fan of `donationMethod` either. It's already an improvement so not a blocker on my side, but I can suggest this alternative name:


```suggestion
    source: `/:collectiveSlug/:verb(donate)/:paymentFlow(crypto)?/:step(${contributionFlowSteps})?`,
```

Reply:
Thanks for the suggestion. I've changed that. 👍🏽